### ISSUE TITLE
load geometry less layer

### ISSUE THREAD

**Issue by 3nids** [opengisch/solocator#10]
waiting on example

**Comment by andreasneumann** [comment]
see separate issue at https://github.com/opengisch/solocator/issues/11

this needs to wait for now, as we have no data sets exposed yet. Will need work from the SoGIS-Team. We can close this ticket and use #11
